lib/create-icon-set-from-fontawesome5.js

```diff
@@ -0,0 +1,165 @@
+import React, { PureComponent } from 'react';
+
+import { PixelRatio, Platform, processColor } from 'react-native';
+
+import PropTypes from 'prop-types';
+
+import createIconSet, {
+  DEFAULT_ICON_COLOR,
+  DEFAULT_ICON_SIZE,
+  ensureNativeModuleAvailable,
+  NativeIconAPI,
+} from '../lib/create-icon-set';
+
+export const FA5Style = {
+  regular: 0,
+  light: 1,
+  solid: 2,
+  brand: 3,
+};
+
+let setupFA5 = true;
```

Comment:
I'm being picky but I don't like this name as to me it reads as a function when out of context. Maybe `wasFA5SupportInitialized`, IDK.